Microsoft.DotNet.Interactive.CSharp/CSharpKernelExtensions.cs

```diff
@@ -57,115 +80,146 @@ public static class CSharpKernelExtensions
             };
 
             var restoreContext = new PackageRestoreContext();
-            
-            command.Handler = CommandHandler.Create<NugetPackageReference, KernelInvocationContext>(async (package, pipelineContext) =>
-            {
-                var addPackage = new AddNugetPackage(package);
 
-                addPackage.Handler = async context =>
+            command.Handler = CommandHandler.Create<PackageReference, KernelInvocationContext>(async (package, pipelineContext) =>
+            {
+                var addPackage = new AddPackage(package)
                 {
-                    var message = $"Installing package {package.PackageName}";
-                    if (!string.IsNullOrWhiteSpace(package.PackageVersion))
+                    Handler = async context =>
                     {
-                        message += $", version {package.PackageVersion}";
+                        await Task.FromResult(
```

Comment:
There's no need for this to be `async`.

Reply:
Unfortunately Handler is defined on KernelBase as:
````
    public delegate Task KernelCommandInvocation(
        KernelInvocationContext context);
````
Therefore in order to avoid the warning:  "async method saying that it lacks 'await' operators"
we need away to add an await.